### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by Jdahn** [qd-cae/qd-eng#5]
Hi there, 

I am trying to install your QD-eng package. 
Unfortunately, I am stuck at installation. 
I pointed the boost library directory to the correct one in the setup.py file. 
I am running on windows with python version 2.7.10.
I tried following commands to install:
```
pip install qd
python -m pip install qd

```
However, both result in following error:
```
Collecting qd
  Could not find a version that satisfies the requirement qd (from versions: )
No matching distribution found for qd
```
How should I proceed to install this package?

**Comment by qd-codie** [comment]
Hey,
for installation with pip on windows you dont need boost or anything else. I precompiled the code already (that's why I build .whl for pip). The message means that it can not find the right precompiled version for your python. Note that the wheels are just for python 64bit distributions (not 32bit)! Also some distributions, even though they are 64bit, require own compilation due to further incompatibilities. 

You now have three choices: 
 - Download Anaconda Python (2.7, 3.5 or 3.6) 64bit and install with pip as described 
 - Tell me your distribution and version and I might compile it for your distribution
 - compile it yourself on windows (I have no description yet)

Of course I recommend you using Anaconda python.

In case of two, tell me your python distribution. Note though that I do not plan on supporting 32bit. If you want me to compile it, I need to know your distribution (open an interactive console and print the header here). E.g I have Anaconda 2.5 - Python 2.7.11 - 64bit.

In case of three I can tell you how to compile it yourself (it is quite simple and you seem to already have downloaded boost). Download and install the [Visual C++ Compiler specifically for python 2](https://www.microsoft.com/en-us/download/details.aspx?id=44266). Then download the whole project *qd-eng* from git. Register boost in the header of the file setup.py (I highlighted it). At last compile and install the package with:  `python2 setup.py install`
